internal/safepath/join_unsafe.go

```diff
@@ -0,0 +1,35 @@
+//go:build !linux && !windows
+// +build !linux,!windows
+
+package safepath
+
+import (
+	"path/filepath"
+
+	"github.com/sirupsen/logrus"
+)
+
+// This is unsafe and suspectible to Time-Of-Check Time-Of-Use attack.
+func Join(path, subpath string) (*SafePath, error) {
```

Comment:
I think this should just error out.

Reply:
The unsafe implementation is removed in latter commit.